gemfiles/rails4.0.gemfile.lock

```diff
@@ -1,60 +0,0 @@
-PATH
```

Comment:
relock instead of deleting / make travis delete them ?
<-> http://grosser.it/2015/08/14/check-in-your-gemfile-lock/